components/cam/src/physics/cam/micro_p3.F90

```diff
@@ -281,8 +281,8 @@ SUBROUTINE p3_init_b()
           do kk = 1,10000
 
              dia = (real(kk)*dd-dd*0.5_rtype)*1.e-6_rtype  ! size bin [m]
-             amg = piov6*997._rtype*dia**3           ! mass [kg]
-             amg = amg*1000._rtype                   ! convert [kg] to [g]
+             amg = piov6*rhow*dia**3           ! mass [kg]  ! AaronDonahue: does this mean that density is hard-coded?  Changing 997 in this expression to rhow from p3_utils
```

Comment:
I think this is ok - the density of water doesn't change much.

Reply:
check, I figured as much but wanted to make sure.